tests/e2e/config/bootstrap.js

```diff
@@ -74,8 +74,8 @@ const pageEvents = [];
 
 // The Jest timeout is increased because these tests are a bit slow
 jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
-// Set default timeout for individual expect-puppeteer assertions. (Default: 500)
-setDefaultOptions( { timeout: EXPECT_PUPPETEER_TIMEOUT || 500 } );
+// Set default timeout for individual expect-puppeteer assertions. (Default: 1000)
+setDefaultOptions( { timeout: EXPECT_PUPPETEER_TIMEOUT || 1000 } );
```

Comment:
For some reason, the E2E WP 5.2 workflow started failing quite regularly on this branch, as can be seen here: https://github.com/google/site-kit-wp/actions/workflows/e2e-tests-wp-5-2.yml?query=branch%3Ainfra%2F1769-upgrade-wordpress-data

The failures are all in the same area, waiting for the page to navigate to the main Dashboard, e.g. https://github.com/google/site-kit-wp/actions/runs/3750192482/jobs/6370581519#step:12:267

![image](https://user-images.githubusercontent.com/18395600/208967763-9e9fdac3-81f6-4afe-b576-13eac78c8088.png)

I was able to reproduce this locally, and found increasing the default assertion timeout allowed the tests to run without fail.

I think we'd be well off increasing this globally here to help make our tests a bit more robust, rather than on a per-test or per-assertion basis.